Update AddOrgModal.js

maxskewes opened the pull request:
Centered form, centered modal, added effect, added click anywhere close
resolves #605

bruceplai reviewed (changes requested):
The visual updates look good.
As far as styling goes, we're trying to move the style={} properties out of the components and into makeStyles or styles.js (which ever you think would be cleaner).  The exception to this rule might be if it's a material UI component that has a built in property that affects style.  For example, Grid has a justify property that might be able to replace style={{ justifyContent: 'center' }}

Also the form the form doesn't do any validation for missing/invalid fields, but I'm guessing that'll be covered in a separate issue?

maxskewes commented:
Yeah Bruce, I thought that the error messages would be a separate issue. Regarding the styling, I had initially (several figma writings ago) put the styling in the style.js but was told not to put anything that was specific to only one file in there, which I think makes sense. I know that with all the different sizing at different breakpoints that was later asked to be written into the code, it's made for a longer styling section to the file than most (if not all) of the other components. But I think this component in an anomaly in that the only real way to get the styling out of the component file is to follow the latest figma and write it as a page with its own styling file, which we decided last meeting not to do. But let's discuss it at the next meeting and try to get a firm consensus as what we're trying to do with this component and it's awesome of you working with me to get this up and functioning.

maxskewes commented:
@bruceplai I created a folder for the AddOrgModal which now houses the style sheet, the index, and the CountrySelector component used in the form. I also added a scroll bar and changed the TextField sizes to 'small' across all display sizes. I think this should close the issue and then I'll open an issue for the error messages and get working on that.

bruceplai reviewed (approved):
Looks good.  Files and style are more organized under AddOrgModal
